client/js/components/procedure/basicSettings/DpEmailList.vue

```diff
@@ -142,25 +145,18 @@ export default {
       this.$emit('saved', extraEmailAddress)
     },
 
+    showUpdateForm (index) {
+      this.formFields.mail = this.emails[index].mail
+      this.itemIndex = index
+    },
+
     updateExtraEmailAddress (index, extraEmailAddress) {
```

Comment:
Is this event handled anywhere?

Reply:
nope e13ad3e588c43000eb5bbcbd90664bc9919e7c04